lib/matplotlib/backend_bases.py

```diff
@@ -2440,6 +2477,19 @@ def stop_event_loop_default(self):
         """
         self._looping = False
 
+    def destroy(self):
+        pass
+
+    @property
+    def manager(self):
+        if self._manager is not None:
+            return self._manager()
+
+    @manager.setter
+    def manager(self, manager):
+        if manager is not None:
+            self._manager = weakref.ref(manager)
```

Comment:
I think I prefer 1. because adding a property shouldn't count as breaking BC (somehow it just feels like a break in BC).